spec/lib/miq_automation_engine/miq_ae_domain_spec.rb

```diff
@@ -3,12 +3,14 @@
 
 describe MiqAeDomain do
   before do
+    EvmSpecHelper.local_guid_miq_server_zone
+    @root_tenant = Tenant.root_tenant
     setup_model
   end
 
   def setup_model
     yaml_file = File.join(File.dirname(__FILE__), 'data', 'domain_test.yaml')
-    import_options = {'yaml_file' => yaml_file, 'preview' => false, 'domain' => '*'}
+    import_options = {'yaml_file' => yaml_file, 'preview' => false, 'domain' => '*', 'tenant_id' => @root_tenant.id}
```

Comment:
@mkanoor ooh - yea. I'm good with that.

I thought the code defaulted to the root tenant if there was no tenant in the import options. So I was thinking that you would have not assigned the tenant in at least 1 of the tests